rust/db_handling/src/identities.rs

```diff
@@ -891,7 +891,7 @@ fn populate_addresses(
         if let Ok(prep_data) = create_address(
             database,
             &address_prep,
-            &network_specs.specs.path_id,
+            "",
```

Comment:
Please consider adding a test for this change

Reply:
believe me I am changing the whole test base atm.